data/json/monsters.json

```diff
@@ -1480,10 +1480,10 @@
     "flags": [ "DIGS", "HEARS", "POISON", "GOODHEARING", "BASHES", "DESTROYS", "WARM", "LEATHER", "FAT" ]
   },
   {
-    "id": "mon_gracke",
+    "id": "mon_gracken",
```

Comment:
You should not change id of monster - I am not sure if we have a migration mechanism for monsters, also if change you change it, you should also change ids in c++ code:

```
C:\Projects\Cataclysm-DDA\src\field.cpp (1 hit)
	Line 1407:                             mtype_id( "mon_flaming_eye" ), mtype_id( "mon_kreck" ), mtype_id( "mon_gracke" ),
  C:\Projects\Cataclysm-DDA\src\mapgen.cpp (3 hits)
	Line 71: const mtype_id mon_gracke( "mon_gracke" );
	Line 71: const mtype_id mon_gracke( "mon_gracke" );
	Line 3379:                     mon_flaming_eye, mon_kreck, mon_gracke, mon_blank, mon_gozu, mon_shoggoth,
  C:\Projects\Cataclysm-DDA\src\map_extras.cpp (8 hits)
	Line 33: static const mtype_id mon_gracke( "mon_gracke" );
	Line 33: static const mtype_id mon_gracke( "mon_gracke" );
	Line 111:             mon_gelatin, mon_mi_go, mon_kreck, mon_gracke,
	Line 140:             mon_gelatin, mon_mi_go, mon_kreck, mon_gracke
	Line 176:             mon_gelatin, mon_mi_go, mon_kreck, mon_gracke
	Line 384:             mon_gelatin, mon_mi_go, mon_kreck, mon_gracke
	Line 440:             mon_gelatin, mon_flaming_eye, mon_kreck, mon_gracke, mon_blank
	Line 524:             mon_gelatin, mon_flaming_eye, mon_kreck, mon_gracke, mon_blank
```